airgun/entities/computeresource.py

```diff
@@ -0,0 +1,74 @@
+from navmazing import NavigateToSibling
+from airgun.entities.base import BaseEntity
+from airgun.navigation import NavigateStep, navigator
+from airgun.views.computeresource import (
+    ComputeResourcesView,
+    ResourceProviderDetailsView,
+)
+
+
+class ComputeResourceEntity(BaseEntity):
+
+    def create(self, values):
+        view = self.navigate_to(self, 'New')
+        view.fill(values)
+        if 'provider' in values.keys() and values['provider'] == 'oVirt' \
```

Comment:
we are going to think about that block and whether conditional views can use `after_fill()` method

Reply:
Did you agree on something? Because now I'm just waiting for your opinion to move to issue #88. Should I change anything else here or in tests?

Reply:
yes, I tested and solution works just perfect, let's me describe it for you. That should be last step I believe for that PR

Reply:
Please, remove
```
if 'provider' in values.keys() and values['provider'] == 'oVirt' \
+                and 'provider_content.certification_authorities' not in \
+                values.keys():
+            view.provider_content.load_datacenters.click()
+        if 'provider' in values.keys() and values['provider'] == 'EC2':
+            view.provider_content.load_regions.click()
+        if 'provider' in values.keys() and values['provider'] == 'VMware':
+            view.provider_content.load_datacenters.click()
```
and then add corresponding block to each provider you need:
```
+    @provider_content.register('oVirt')
+    class oVirtProviderForm(View):
+        url = TextInput(id='compute_resource_url')
+        user = TextInput(id='compute_resource_user')
+        password = TextInput(id='compute_resource_password')
+        load_datacenters = Text("//*[contains(@id,'test_connection_button')]")
+        certification_authorities = TextInput(id='compute_resource_public_key')
     
          def after_fill(self, was_change):
              self.load_datacenters.click()
```
```
+    @provider_content.register('EC2')
+    class EC2ProviderForm(View):
+        http_proxy = TextInput(id='compute_resource_http_proxy_id')
+        access_key = TextInput(id='compute_resource_user')
+        secret_key = TextInput(id='compute_resource_password')
+        load_regions = Text("//*[contains(@id,'test_connection_button')]")

          def after_fill(self, was_change):
              self.load_regions.click()
```
```
+    @provider_content.register('VMware')
+    class VMwareProviderForm(View):
+        vcenter = TextInput(id='compute_resource_server')
+        user = TextInput(id='compute_resource_user')
+        password = TextInput(id='compute_resource_password')
+        display_type = Select(id='compute_resource_display_type')
+        vnc_console_passwords = Checkbox(
+            id='compute_resource_set_console_password')
+        enable_caching = Checkbox(id='compute_resource_caching_enabled')
+        load_datacenters = Text("//*[contains(@id,'test_connection_button')]")

          def after_fill(self, was_change):
              self.load_datacenters.click()
```

Reply:
That shouldn't affect test logic as far as I see